packages/storybook/src/clinical/organisms/patient-banner/patient-banner.test.tsx

```diff
@@ -1,10 +1,17 @@
-import { render } from '@testing-library/react'
+import { render, screen } from '@testing-library/react'
 
 import PatientBanner from '@ltht-react/patient-banner'
-import { DeceasedPatient } from './patient-banner.fixtures'
+import { AlivePatient, DeceasedPatient, groupsIncludingPedigreeGroup } from './patient-banner.fixtures'
 
 describe('Patient Banner', () => {
   it('Renders', () => {
     render(<PatientBanner patient={DeceasedPatient} />)
   })
+
+  it('Shows Pedigree Number if one is provided', () => {
+    render(<PatientBanner patient={AlivePatient} patientGroups={groupsIncludingPedigreeGroup} />)
+
+    expect(screen.getByText('Ped No.')).toBeInTheDocument()
+    expect(screen.getByText('PED048164')).toBeInTheDocument()
```

Comment:
Unfortunately, Jest Dom doesn't support testing media queries. Since all secondary information is hidden (well, `display: none`) on windows smaller than 768px in width, the `toBeVisible()` assertion will fail. 
Ideally we need to test this in something like Cypress or Testcafe.

Reply:
Sorry, this is unclear.. leave the address on the left, wrap everything else and have it appear on the right side with some margins so it matches the primary info in the purple bit as discussed :)